bigchaindb/db/utils.py

```diff
@@ -29,7 +29,6 @@ def create_database(conn, dbname):
     logger.info('Create database `%s`.', dbname)
     r.db_create(dbname).run(conn)
 
-
```

Comment:
You can look for an add-on to your text editor to check that your code follows PEP 8 conventions. Or you can use the pep8 tool directly as documented in https://pypi.python.org/pypi/pep8.

For example, for a missing blank line as noted above you would see an error like:


```bash
 bigchaindb/db/utils.py|32 col 1 C| E302 expected 2 blank lines, found 1 [pep8]
```